test/fixtures/block-element-grouping.expected.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
+    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" jcr:title="Sites Franklin Example" sling:resourceType="core/franklin/components/page/v1/page">
+        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
+            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
+                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Teaser Grouping" image="/content/dam/teaser-background.png" imageAlt="A group of people sitting on a stage" teaserText_subtitle="Adobe Experience Cloud" teaserText_title="Meet the Experts" teaserText_titleType="h2" teaserText_description="Join us in this ask me everything session ..." teaserText_cta1="More Details" teaserText_cta1Text="More Details" teaserText_cta1Title="" teaserText_cta2="RSVP" teaserText_cta2Type="primary" teaserText_cta2Text="RSVP" teaserText_cta2Title="" model="teaser-grouping"/>
+                <block_0 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Teaser Grouping" image="/content/dam/teaser-background.png" imageAlt="A group of people sitting on a stage" teaserText_title="Meet the Experts" teaserText_titleType="h2" teaserText_description="Join us in this ask me everything session ..." teaserText_cta1="More Details" teaserText_cta1Text="More Details" teaserText_cta1Title="" teaserText_cta2="RSVP" teaserText_cta2Type="primary" teaserText_cta2Text="RSVP" teaserText_cta2Title="" model="teaser-grouping"/>
+                <block_1 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Teaser Grouping" image="/content/dam/teaser-background.png" imageAlt="A group of people sitting on a stage" teaserText_subtitle="Adobe Experience Cloud" teaserText_title="Meet the Experts" teaserText_titleType="h2" teaserText_description="&lt;p>Join us in this ask me everything session ...&lt;p>&lt;p>... and aks your questions.&lt;/p>" teaserText_cta1="More Details" teaserText_cta1Text="More Details" teaserText_cta1Title="" teaserText_cta2="RSVP" teaserText_cta2Type="primary" teaserText_cta2Text="RSVP" teaserText_cta2Title="" model="teaser-grouping"/>
```

Comment:
shouldn't it be 
```&lt;p>Join us in this ask me everything session ...&lt;/p>&lt;p>... and aks your questions.&lt;/p>```
instead of 
```&lt;p>Join us in this ask me everything session ...&lt;p>&lt;p>... and aks your questions.&lt;/p>```